src/partials/device_selection.hbs

```diff
@@ -31,40 +31,70 @@
-              <div class="tile-button__title">{{device.type}}</div>
-            {{/ifEq}}
+          {{#ifEq device.type "Android"}}
+          <div class="tile-button__title">PingOne Authenticator</div>
+          {{else ifEq device.type "iPhone"}}
+          <div class="tile-button__title">PingOne Authenticator</div>
+          {{else ifEq device.type "TOTP"}}
+          <div class="tile-button__title">Authenticator App</div>
+          {{else ifEq device.type "SMS"}}
+          <div class="tile-button__title">Text Message</div>
+          {{else ifEq device.type "Email"}}
+          <div class="tile-button__title">Email</div>
+          {{else ifEq device.type "SECURITY_KEY"}}
+          <div class="tile-button__title">Security Key</div>
+          {{else ifEq device.type "PLATFORM"}}
+          <div class="tile-button__title">Biometrics</div>
+          {{else}}
+          <div class="tile-button__title">{{device.type}}</div>
+          {{/ifEq}}
           {{/if}}
           {{#ifEq device.type "Android"}}
-            <div class="tile-button__content">{{device.name}}</div>
+          <div class="tile-button__content">{{device.name}}</div>
           {{else ifEq device.type "iPhone"}}
-            <div class="tile-button__content">{{device.name}}</div>
+          <div class="tile-button__content">{{device.name}}</div>
           {{else if device.target}}
-            <div class="tile-button__content">{{device.target}}</div>
+          <div class="tile-button__content">{{device.target}}</div>
           {{/ifEq}}
+          {{#if device.defaultDevice}}
+          <div class="device-selector__item-status"><span class="muted-text">Default</span></div>
+          {{/if}}
+        </div>
+      </div>
+      <div class="device-selector__item-controls" id="kebab-menu-icon-id" data-mfa-selection-kebab-menu-container="{{device.id}}">
+        <div class="popover popover--left" data-id="popover-nav-menu">
+          <a class="popover__trigger" data-id="popover-nav-menu-trigger">
+            <div>
+              <button class="button file-input~~button popover-menu__control" data-id="button" style="border:none" type="button">
+                <svg id="kebab-menu-svg-id" viewBox="0 0 24 24" role="presentation" style="width: 26px; height: 26px;">
+                  <path
+                    d="M12,16A2,2 0 0,1 14,18A2,2 0 0,1 12,20A2,2 0 0,1 10,18A2,2 0 0,1 12,16M12,10A2,2 0 0,1 14,12A2,2 0 0,1 12,14A2,2 0 0,1 10,12A2,2 0 0,1 12,10M12,4A2,2 0 0,1 14,6A2,2 0 0,1 12,8A2,2 0 0,1 10,6A2,2 0 0,1 12,4Z"
+                    style="fill: rgb(104, 111, 119);"></path>
+                </svg></button>
+            </div>
+          </a>
+          <div id="device-management-popup-frame-{{device.id}}" data-mfa-selection-kebab-menu-container="{{device.id}}" data-id="popup-frame" data-parent="popover-nav-menu"
+            class="popper-container popover-display" ;
+            style="display:none; position: absolute; will-change: transform; right: 0px;">
```

Comment:
will-change isn't critical right? Is the behavior okay in IE11?

Reply:
copy pasted from P1 UI. Had no idea what it did. Removed now :)